### PR TITLE
update helios links/references in readme

### PR DESCRIPTION
**Pull request by Dhaulagiri**
### :pushpin: Summary

updates helios links/references in readme to better represent the new home for things

### :camera_flash: Screenshots

Screenshot 2023-01-19 at 08 52 23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,14 @@
-# Helios Design System
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/1672302/213473448-ac8cd0f7-cde2-49ef-b8ef-4067677569f2.jpg">
+</p>
+
+<p align="center">The Helios Design System provides the building blocks to design and implement consistent, accessible, and delightful product experiences across HashiCorp.</p>
+
+
```
Comment (Dhaulagiri) [inline README.md:7]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,14 @@
-# Helios Design System
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/1672302/213473448-ac8cd0f7-cde2-49ef-b8ef-4067677569f2.jpg">
+</p>
+
+<p align="center">The Helios Design System provides the building blocks to design and implement consistent, accessible, and delightful product experiences across HashiCorp.</p>
+
```
Comment (Dhaulagiri) [inline README.md:6]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -31,7 +41,7 @@ Flight icons in different formats (SVG/SVG Sprite/React)
 Design tokens
 
 - npm package: `@hashicorp/design-system-tokens`
-- website: [design-system-components-hashicorp.vercel.app/foundations/tokens](https://design-system-components-hashicorp.vercel.app/foundations/tokens)
+- website: [https://helios.hashicorp.design/foundations/tokens](https://helios.hashicorp.design/foundations/tokens)
 - more info: see [tokens/README](packages/tokens/README.md) and [tokens/CONTRIBUTING](packages/tokens/CONTRIBUTING.md) for details on how to update the design tokens.
 
 ### `flight-website`
```
Comment (Dhaulagiri) [inline README.md:47]: Open to the idea of also just removing this section at this point as this site won't exist much longer

File: README.md [hunk 4]
```diff
@@ -1,4 +1,14 @@
-# Helios Design System
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/1672302/213473448-ac8cd0f7-cde2-49ef-b8ef-4067677569f2.jpg">
```
Comment (MelSumner) [inline README.md:2]:
```suggestion
  <img src="https://user-images.githubusercontent.com/1672302/213473448-ac8cd0f7-cde2-49ef-b8ef-4067677569f2.jpg" role="presentation" alt="">
```

Question: Why are we not hosting this image on our own website?

Comment (Dhaulagiri) [reply README.md:2]: > Why are we not hosting this image on our own website

The assets hosted on our website are fingerprinted and therefore would change frequently

Comment (Dhaulagiri) [reply README.md]: added the role in a new commit

**Review by heatherlarsen — APPROVED** [review]
:shipit:

**Pull request merged**